web/client/examples/3dviewer/containers/Viewer.jsx

```diff
@@ -45,7 +45,7 @@ class Viewer extends React.Component {
     static defaultProps = {
         mapOptions: {
             terrainProvider: {
-                type: "cesium",
+                type: "ellipsoid",
                 url: "https://assets.agi.com/stk-terrain/world",
```

Comment:
I know I'm boring, but url and requestVertexNormals are not needed anymore, please leave only "type"

Reply:
of course the same in the other file, please always remove lines of code and configurations that are not needed